.circleci/config.yml

```diff
@@ -64,24 +64,26 @@ jobs:
 workflows:
   build:
     jobs:
-      - lint
-      - test:
-          matrix:
-            parameters:
-              gemfile:
-              - "gemfiles/avro1_11_rails6_1.gemfile"
-              - "gemfiles/avro1_11_rails7_0.gemfile"
-              - "gemfiles/avro1_11_rails7_1.gemfile"
-              ruby-version:
-              - "3.0.6"
-      - test:
-          matrix:
-            parameters:
-              gemfile:
-                - "gemfiles/avro1_11_rails6_1.gemfile"
-                - "gemfiles/avro1_11_rails7_0.gemfile"
-                - "gemfiles/avro1_11_rails7_1.gemfile"
-              ruby-version:
-                - "3.0.6"
-                - "3.1.4"
-                - "3.2.2"
+    - lint
+    - test:
```

Comment:
It looks to me like this is covered by the test step below?

Reply:
huh weird, good catch. Removed the duplicate set!